test/utils/shippable/sanity.sh

```diff
@@ -6,25 +6,35 @@ install_deps="${INSTALL_DEPS:-}"
 
 cd "${source_root}"
 
+# FIXME REPOMERGE: No need to checkout ansible
+build_dir="/tmp/ansible"
+git clone "https://github.com/ansible/ansible.git" "${build_dir}" --recursive
```

Comment:
I'd install a trap after the clone to remove the temporary directory on exit, in case the script is run locally.

```shell
trap 'rm -rf "${build_dir}"' EXIT
```

The rest looks good.